# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ipset/0001-include-sys-types.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/include/libipset/data.h	2013-01-02 00:29:07.395065580 +0100
+++ b/include/libipset/data.h	2013-01-02 00:29:56.944261628 +0100
@@ -8,6 +8,7 @@
 #define LIBIPSET_DATA_H
 
 #include <stdbool.h>				/* bool */
+#include <sys/types.h>				/* size_t */
 #include <libipset/nf_inet_addr.h>		/* union nf_inet_addr */
 
 /* Data options */
